### ISSUE TITLE
💡 How to use Environments + Secrets

### ISSUE THREAD

**Issue by gr2m** [gr2m/helpdesk#6]
follow up to https://mobile.twitter.com/metaskills/status/1386504715903569927, I want to make sure not to forget. It's definitely a topic I'm interested in /cc @metaskills

**Comment by metaskills** [comment]
INDEED! I am making a case now to upgrade our account and having to dig in materials to learn more.
